From bcdd24d811bb6bc5a3232d55f8af11ddd9e3873e Mon Sep 17 00:00:00 2001
From: Xiretza <xiretza@xiretza.xyz>
Date: Sun, 12 Sep 2021 14:06:09 +0200
Subject: [PATCH 3/3] Revert "ci: use yosys from conda"

This reverts part of commit 82d39a92a19badbbf7a7f82c4ac3672e03941697.

Conda is useless and the exact opposite of what we want for packaging.
---
 Makefile | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/Makefile b/Makefile
index 780ff83..a2ec069 100644
--- a/Makefile
+++ b/Makefile
@@ -14,14 +14,6 @@ PLUGINS_TEST := $(foreach plugin,$(PLUGIN_LIST),test_$(plugin))
 
 all: plugins
 
-TOP_DIR := $(realpath $(dir $(lastword $(MAKEFILE_LIST))))
-REQUIREMENTS_FILE ?= requirements.txt
-ENVIRONMENT_FILE ?= environment.yml
-
-include third_party/make-env/conda.mk
-
-env:: | $(CONDA_ENV_PYTHON)
-
 define install_plugin =
 $(1).so:
 	$$(MAKE) -C $(1)-plugin $$@
@@ -44,7 +36,7 @@ install: $(PLUGINS_INSTALL)
 
 test: $(PLUGINS_TEST)
 
-clean:: $(PLUGINS_CLEAN)
+clean: $(PLUGINS_CLEAN)
 
 CLANG_FORMAT ?= clang-format-8
 format:
-- 
2.33.0

